doc/concepts/pipeline-concepts/datum/relationship-between-datums.md

```diff
@@ -0,0 +1,159 @@
+processing speed. You know your data better and can decide
+how to optimize your pipeline based on the repository structure
+and data generation workflows.
+For more information about glob patterns, see
+[Glob Pattern](glob-pattern.html).
+
+Disregarding of how many datums you define and how many
+filesystem objects a datum has, Pachyderm always matches the
+number of input datums with the number of output datums. For
+example, if you have three input datums in `pfs/`, you will
+have three output datums in `pfs/out`. `pfs/out` is the
+output repository that Pachyderm creates automatically for
+each pipeline. You can add your changes in any order and
+submit them in one or multiple commits, the result of your
+pipeline processing remains the same.
+
+Another aspect of Pachyderm data processing is
+appending and overwriting files. By default, Pachyderm
+appends new data to the existing data. For example, if you
+have a file `foo` that is 100 KB in size in the repository `A`
+and add the same file `foo` to that repository again by
+using the `pachctl put file` command, Pachyderm does not
+overwrite that file but appends it to the file `foo` in the
+repo. Therefore, the size of the file `foo` doubles and
+becomes 200 KB. Pachyderm enables you to overwrite files as
+well by using the `--overwrite` flag. The order of processing
+is not guaranteed, and all datums are processed randomly.
+For more information, see [File](file.html).
+
+When new data comes in, a Pachyderm pipeline automatically
+starts a new job. Each Pachyderm job consists of the
+following stages:
+
+1. Creation of input datums. In this stage, Pachyderm breaks
+input files into datums according to the glob pattern setting
+in the pipeline specification.
+1. Transformation. The pipeline uses your code to processes the
+datums.
+1. Creation of output datums. Pachyderm creates file or files from the
+processed data and combines them into output datums.
+1. Merge. Pachyderm combines all files with the same file path
+by appending, overwriting, or deleting them to create the final commit.
+
+If you think about this process in terms of filesystem objects and
+processing abstractions, the following transformation happens:
+
+**input files = > input datums => output datums => output files**
+
+This section provides examples that help you understand such fundamental
+Pachyderm concepts as the datum, incremental processing, and phases of
+data processing.
+
+## Example 1: One file in the input datum, one file in the output datum
+
+The simplest example of datum processing is when you have one file in
+the input datum that results in one file in the output datum.
+In the diagram below, you can see three input datums, each of which
+includes one file, that result in three output datums. Whether you have
+submitted all these datums in a single or multiple commits, the final
+result remains the same—three datums, each of which has one file.
+
+In the diagram below, you can see the following datums:
```

Comment:
Remove:

In the diagram below, you can see the following datums:

  - `datum 1` has one file and results in one file in one output datum.
 - `datum 2` has one file and results in one file in one output datum.
 - `datum 3` has one file and results in one file in one output datum.

It's already covered my my addition above. We can just have the diagram now.